blocks/dashboard/renderAccount.js

```diff
@@ -27,8 +28,8 @@ export default async function renderAccount({ container, nav }) {
         <span title="${user.email}">${user.email}</span>
     </div>
     <div>
-        <strong>Last update</strong>
-        <span>${new Date(user.updated_at).toLocaleDateString()}</span>
+        <strong>Last login</strong>
```

Comment:
IIRC we said we're going to display something like "membership since" using the `created_at` property.

Reply:
yeah I'll do that